### ISSUE TITLE
Pcons temporary changes in explorable area

### ISSUE THREAD

**Issue by UltraPower31** [HasKha/GWToolboxpp#550]
It would be useful for me if you add a checkbox to the pcons window that makes changes to it only temporary for the time I am in the explorable area and reset to the settings I had in the outpost after resigning and zoning back there.
For example when I play TT in DoA and it's a city start I can use a cupcake after city is done for the rest of the run. I wanna auto use the cupcake when it runs out only in this run. When I resign and zone back, the cupcake should be disabled again.
So basically when the checkbox is ticked and you enable or disable pcons those settings are only saved when you are in an outpost and auto reset to those settings when you go back from an explorable area to an outpost.

**Comment by 3vcloud** [comment]
There are a few ways around this - you could set up your pcons to load with your build, you could set pcons to be disabled when re-entering an outpost so you'd have to review your pcons before you enter an area, or you could just get into the habit of not having cupcakes on until you've either done city or have loaded in. Sounds like any of these 3 options would do want you want with the same if not less amount of effort than having to keep an eye on a tickbox in the pcons window

**Comment by PostCrafter** [comment]
Neither of the presented workarounds are really satisfying:
- I'm generally not reloading my build every time I go into an outpost, so my pcon settings won't be reloaded either.
- Disabling and reviewing your pcons sounds weird to me as well, I want them to either be on all the time or I'll just hit my toggle hotkey and they're back on again.
- Having to remember this it is likely the reason why this is a request in the first place, it happens farily often that you toggle them on during the run and just forget about it until someone zones into the next run.

Having "instance pcons" is not only useful for TT in DoA, but can also be useful for LT and T2 in UW where you don't want cupcakes to be active until a certain point in the run. If a run fails and someone zones back in immediately you're stuck having to do awkward pulls just because you didn't remember to turn it off in those few seconds.

**Comment by 3vcloud** [comment]
What about chat command to set a pcon on/off/toggle?

**Comment by PostCrafter** [comment]
I like the idea of toggling pcons individually instead of relying on use item hotkeys or UI interaction, but I don't think it solves the issue here.

**Comment by 3vcloud** [comment]
1 solution for many scenarios - make a cupcake off hotkey without a binding, set it to trigger on map change

**Comment by UltraPower31** [comment]
I mean if that's less complicated to implement than my request sure, would be cool if you could implement that.
